tests/scripts/thread-cert/Cert_5_5_04_SplitMergeRouters.py

```diff
@@ -108,11 +115,45 @@ def test(self):
 
         self.assertEqual(self.nodes[LEADER].get_state(), 'router')
 
+        self.collect_ipaddrs()
         addrs = self.nodes[ROUTER4].get_addrs()
         for addr in addrs:
             if addr[0:4] != 'fe80':
                 self.assertTrue(self.nodes[ROUTER3].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        ROUTER_3 = pv.vars['ROUTER_3']
+        ROUTER_4 = pv.vars['ROUTER_4']
+        leader_pkts = pkts.filter_wpan_src64(LEADER)
+        router1_pkts = pkts.filter_wpan_src64(ROUTER_1)
+
+        # Step 2: The Leader  MUST send properly formatted MLE Advertisements
+        router1_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+        leader_pkts.range(router1_pkts.index).filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: {0, 11, 9} == set(p.mle.tlv.type))
+
+        router1_pkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        lreset_start = router1_pkts.index
+
+        router1_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
+        lreset_stop = router1_pkts.index
+
+        # Step 3: The Leader MUST stop sending MLE advertisements.
+        leader_pkts.range(lreset_start, lreset_stop).filter_mle_cmd(MLE_ADVERTISEMENT).must_not_next()
+
+        # Step 5: Router_4 MUST send an ICMPv6 Echo Reply to Router_3
```

Comment:
Is there a Step 4?

Reply:
Step 4 is an operation for making Create two partitions after leader is removed and NETWORK_ID_TIMEOUT, should we add the step 4 description?